conferences/2019/javascript.json

```diff
@@ -309,5 +309,14 @@
     "city": "Budapest",
     "country": "Hungary",
     "twitter": "@jsconfbp"
+  },
+  {
+    "name": "React India 2019",
```

Comment:
Can you just remove the `2019`? We try not to put these in the names since we have the date already. :) 
Thanks! 🙏